<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame
{
   background-color: rgba( 230, 230, 216, 90% );
	border-radius:10px;
   
}
#Form
{
background:url(&quot;./BG.jpg&quot;);
}
QLineEdit
{
background:transparent;
border:none;
border-bottom: 1px solid black;
}
QPushButton
{
	background:#B0B0B0;
	border-radius:15px;
	cursor:pointer;
}

QPushButton:Hover
{
	background:#918D8D;
	border-radius:15px;
	cursor:pointer;
}
#print
{
	background:#95a4cf;
	border-radius:10px;
}
QGroupBox
{
background-color: rgba( 230, 230, 216, 90% );
border-radius:10px;
}
QLabel
{
background-color: rgba( 0, 0, 0, 0% );
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>481</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="subBtn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Subdomain</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>2435355-200.png</normaloff>2435355-200.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="sqlBtn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SQL Injection</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>3358758-200.png</normaloff>3358758-200.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="dosBtn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DOS</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>1464646-200.png</normaloff>1464646-200.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="xssBtn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>XSS</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>XSS-file-program-512.png</normaloff>XSS-file-program-512.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>511</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Web Pentesting</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
